### PR TITLE
Fixing the typos and grammatical errors in Readme files

### PR DESCRIPTION
**Pull request by mohitm15**
**What kind of change does this PR introduce?**

Bug Fix, fixes the issue #1245 

**If relevant, did you update the documentation?**
Yes, Security.md and webpack-scaffold/readme.md

**Summary**
Fixing the typos and grammatical errors in Readme files

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

**Comment by mohitm15** [comment]
I uses ` git commit -m (message)`

**Comment by anshumanv** [comment]
We have specific commit guidelines which are most easily to adhere to using the commit helper package that we use, just do `npm run commit` and follow the prompt then you should be good.

File: packages/webpack-scaffold/README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ this.configuration.myScaffold.webpackOptions.entry = createDynamicPromise(["app.
 
 Param: `String`
 
-Used to create an [assetFilterFunction](https://webpack.js.org/configuration/performance/#performance-assetfilter)
+Used to create a [assetFilterFunction](https://webpack.js.org/configuration/performance/#performance-assetfilter)
```
Comment (jamesgeorge007) [inline packages/webpack-scaffold/README.md]: The former was fine.

File: packages/webpack-scaffold/README.md [hunk 2]
```diff
@@ -86,7 +86,7 @@ this.configuration.myScaffold.webpackOptions.entry = createDynamicPromise(["app.
 
 Param: `String`
 
-Used to create an [assetFilterFunction](https://webpack.js.org/configuration/performance/#performance-assetfilter)
```
Comment (snitin315) [inline packages/webpack-scaffold/README.md]: I think `an` is correct.

**Comment by anshumanv** [comment]
Also please sign the CLA.

File: packages/webpack-scaffold/README.md (hunk 1 above)
Comment (mohitm15) [reply packages/webpack-scaffold/README.md]: See its readme context

> The former was fine.

Screenshot 2020-02-22 13:28:31

File: packages/webpack-scaffold/README.md (hunk 2 above)
Comment (mohitm15) [reply packages/webpack-scaffold/README.md]: See above comment, self-explanatory

File: packages/webpack-scaffold/README.md (hunk 1 above)
Comment (jamesgeorge007) [reply packages/webpack-scaffold/README.md]: @mohitm15 you should be making the change [here](https://github.com/webpack/webpack-cli/tree/next/packages/webpack-scaffold#createexternalfunction)

Comment (jamesgeorge007) [reply packages/webpack-scaffold/README.md]: Instead you made the change here:
Screenshot from 2020-02-22 14-54-43

Comment (mohitm15) [reply packages/webpack-scaffold/README.md]: Thanks for correcting @jamesgeorge007

**Comment by mohitm15** [comment]
> We have specific commit guidelines which are most easily to adhere to using the commit helper package that we use, just do `npm run commit` and follow the prompt then you should be good.

Getting error logs while running it

**Comment by anshumanv** [comment]
> Getting error logs while running it

What errors? Trace?

**Comment by mohitm15** [comment]
> > Getting error logs while running it
> 
> What errors? Trace?

Screenshot 2020-02-22 15:58:17

**Comment by anshumanv** [comment]
Did you install the project dependencies?

**Comment by ematipico** [comment]
`commit` a command of our CLI :)

Maybe you wanted to do `git commit`

**Comment by mohitm15** [comment]
> Did you install the project dependencies?
NO , can you direct me to the link of project setup on local machine

**Comment by anshumanv** [comment]
@mohitm15 - https://github.com/webpack/webpack-cli/blob/next/.github/CONTRIBUTING.md

Read the Docs 😄

**Comment by rishabh3112** [comment]
Thanks @mohitm15

**Comment by mohitm15** [comment]
> @mohitm15 - https://github.com/webpack/webpack-cli/blob/next/.github/CONTRIBUTING.md
> 
> Read the Docs

@anshumanv  I am facing a problem regarding comminting the changes for my next PR. Can we have chat over gitter channel

**Comment by anshumanv** [comment]
> @anshumanv I am facing a problem regarding comminting the changes for my next PR. Can we have chat over gitter channel

Sure we can, what's the problem exactly? Doing a discussion here will also help anyone who is having similar issues so I'd recommend it doing here.

**Comment by mohitm15** [comment]
> > @anshumanv I am facing a problem regarding comminting the changes for my next PR. Can we have chat over gitter channel
> 
> Sure we can, what's the problem exactly? Doing a discussion here will also help anyone who is having similar issues so I'd recommend it doing here.

Sure

I had made changes in a separate branch for my latest issue, but after adding the changes using `git add <filename>` , I run `git commit -m <message>` but its not working, so I then tried yarn commit, but still causing error

**Comment by anshumanv** [comment]
> I had made changes in a separate branch for my latest issue, but after adding the changes using git add <filename> , I run git commit -m <message> but its not working, so I then tried yarn commit, but still causing error

What's the error? Are you sure you did yarn install?

**Comment by mohitm15** [comment]
> > I had made changes in a separate branch for my latest issue, but after adding the changes using git add  , I run git commit -m  but its not working, so I then tried yarn commit, but still causing error
> 
> What's the error? Are you sure you did yarn install?

Yes

**Comment by snitin315** [comment]
refer [https://github.com/webpack/webpack-cli/pull/1267](https://github.com/webpack/webpack-cli/pull/1267) .

Make sure your branch is up-to-dat with upstream and have this commit .

**Pull request merged**